### ISSUE TITLE
Documentation versioning

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#11]
### Description

Let's put in place the versioning of the documentation.

**Comment by pylapp** [comment]
We won't make versioning of the documentation.

In fact, we use the official Apple tool which will generate thousands of files. For example, with our current code base we got up to 6,000 files for a ZIP archive of 17 MB. Storing all thse files in folders dedicated to old versions, in the same way as ODS, won't be useful. It will increase for nothing the size of the branch, making it heavier and heavier to update and use.

We will still put the ZIP archive of the HTML documentation and the Xcode doc archive as artifacts  of each release so that users (developers used to use their IDE) will get (if they want) quickly the version for the release they want.

In a nutshell, we make things more simple and efficient :)

cc @ludovic35
